cmd/kritis/signer/main.go

```diff
@@ -34,9 +35,10 @@ import (
 )
 
 func main() {
-	var image, json_key_path, pri_key_path, passphrase, pub_key_path, policy_path string
+	var image, timeout_str, json_key_path, pri_key_path, passphrase, pub_key_path, policy_path string
 
 	flag.StringVar(&image, "image", "", "image url, e.g., gcr.io/foo/bar@sha256:abcd")
+	flag.StringVar(&timeout_str, "timeout", "5m", "timeout for checking on image vulnerability , e.g., 600s, 5m")
```

Comment:
Please give it some more descriptive name.  containeranalysis_timeout, vulnz_timeout, or something else